bin/improver-probabilities-to-realizations

```diff
@@ -45,26 +47,100 @@ def main():
         (['--no-of-realizations'],
```

Comment:
Could we use underscores here please, in line with the general convention.

Reply:
Done